### PR TITLE
Add package.json

### PR DESCRIPTION
**Pull request by tali**
For https://gerrit-review.googlesource.com/c/gerrit/+/322884 I am working on building an own hljs package using rollup.
For this to work, I have to be able to manage this package with yarn, so let's add a `package.json` file.

### CODE REVIEW

File: src/languages/soy.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ Contributors: Chema Balsas <jbalsas@gmail.com>
 Description: A client- and server-side templating system that helps you dynamically build reusable HTML and UI elements
 Category: template
 */
-export default function(hljs) {
+module.exports = function(hljs) {
```
Comment (tali) [inline src/languages/soy.js]: I'll rework my rollup configuration so that we can keep the ES6 module here.

File: package.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+        "highlightjs",
+        "syntax-highlight",
+        "syntax"
+    ],
+    "author": "Chema Balsas",
+    "license": "BSD 3-clause",
+    "devDependencies": {
+    },
+    "dependencies": {
+        "highlight.js": "^10.4.1"
```
Comment (joshgoebel) [inline package.json]: 10.4 is no longer supported, so I'd suggesting pinning against 11.

Comment (tali) [reply package.json]: Well at least Gerrit does not yet work with 11 (https://gerrit-review.googlesource.com/c/gerrit/+/322890).
So I'd at least like to keep it compatible with 10.

Comment (tali) [reply package.json]: Maybe we could specify `"^11.3.1 || ^10.4.1"`?

Comment (joshgoebel) [reply package.json]: That sounds nice if that actually works.

Comment (tali) [reply package.json]: At least https://classic.yarnpkg.com/lang/en/docs/dependency-versions/#toc-unions describes that syntax and it seams to work with yarn in Gerrit.

**Comment by tali** [comment]
Ping :-)

**Comment by joshgoebel** [comment]
@paladox Are you no longer maintaining?

**Comment by paladox** [comment]
> @paladox Are you no longer maintaining?

I missed the latest changes, whoops, sorry!

**Pull request merged**